policyengine_us/parameters/gov/states/az/tax/income/credits/property_tax/income_sources.yaml

```diff
@@ -0,0 +1,16 @@
+description: Arizona accounts for the following income sources for the property tax credit.
+values:
+  2021-01-01:
+    - capital_gains_excluded_from_taxable_income
+    - tax_exempt_interest_income
+    - public_pension_income
+    - adjusted_gross_income
```

Comment:
The tax form doesn't quite follow this, and the law says:
>(a) Adjusted gross income as defined by the department.

Does `the department` mean IRS or AZ? Or does AZ not have its own AGI?

Reply:
That law indicates we'll need to create an `az_agi` variable at some point, but I'm specifically curious about which AGI concept this credit considers: federal or state.